<launch>
    
    <rosparam command="load" file="$(find spice_up_coordinator)/config/spice_up_coordinator.yaml" />


    <node pkg="spice_up_coordinator" type="spice_up_action_server.py" name="spice_up_action_server" output="screen"/>
    

</launch>
